### PR TITLE
yoav -  Bag funnel

### PR DESCRIPTION
**Pull request by bossyshrimp68**

### CODE REVIEW

**Review by aaa-png — APPROVED** [review]
ling lang

File: src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/BagFunnel/BagFunnelConstants.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package edu.greenblitz.robotName.subsystems.shooter.Funnel.BagFunnel;
+
+public class BagFunnelConstants {
+
+    public static final int MOTOR_ID = 11;
+
+    public static final double DEBOUNCE_TIME_FOR_LIMIT_SWITCH = 0.5;
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/BagFunnel/BagFunnelConstants.java:7]: Unit

**Review by yonnbro — COMMENTED** [review]
Do not merge, I hate the thought of using a bag motor and I’m not sure we will use it for the funnel

File: src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/BagFunnel/BagFunnelConstants.java (hunk 1 above)
Comment (yonnbro) [reply src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/BagFunnel/BagFunnelConstants.java:7]: unit

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**